apps/em/aat/base/kustomization.yaml

```diff
@@ -3,7 +3,6 @@ kind: Kustomization
 namespace: em
 resources:
   - ../../base
-  - ../../em-icp/em-icp.yaml
```

Comment:
This is missing in https://github.com/hmcts/cnp-flux-config/blob/a092eb4e74e73eee02889bdae2e166d4d936a43c/apps/em/base/kustomization.yaml ?